recipes/taglib/all/conanfile.py

```diff
@@ -45,6 +67,18 @@ def layout(self):
 
     def requirements(self):
         self.requires("zlib/[>=1.2.11 <2]")
+        if self._is_v2:
```

Comment:
Please, add in PR description why do your need Conan 2.x condition in this recipe. It's not clear.

Reply:
It's not for Conan v2, but the unreleased v2 of the package itself. As suggested by @RubenRBS above, I did not include the 2.0-beta release in the package as it will get a stable release in a couple of weeks, likely. I kept the recipe tweaks for that.